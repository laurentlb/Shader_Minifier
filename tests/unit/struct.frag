struct Foo {
  int field1;
  int field2;
  int ab;
  int st;
};

void main() {
  Foo f;
  f.field1 = 1;
  f.field2 = 2;
  f.ab = 3;
  f.st = 4;
}
