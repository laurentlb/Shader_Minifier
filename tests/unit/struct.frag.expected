struct Foo{int field1;int field2;int wz;int xy;};
void main()
{
  Foo i;
  i.field1=1;
  i.field2=2;
  i.wz=3;
  i.xy=4;
}
